src/views/dialogs/NewMarketMessageDetailsDialog.tsx

```diff
@@ -341,7 +342,19 @@ const $Tabs = styled(ToggleGroup)`
   overflow-x: auto;
 ` as typeof ToggleGroup;
 
-const $Details = tw(Details)`[--details-item-height:1.5rem]`;
+const $Details = styled(Details)`
+  ${tw`[--details-item-height:1.5rem]`}
```

Comment:
lol at this point we might as well expand it to normal css but I'm okay either way. its kinda cute